dvc/remote/ssh/__init__.py

```diff
@@ -43,7 +44,7 @@ def __init__(self, repo, config):
             or parsed.username
             or getpass.getuser()
         )
-        self.prefix = parsed.path or "/"
+        self.prefix = url2pathname(parsed.path) or "/"
```

Comment:
Hm, but if you pass ntpath here, then everything will break down the line since we use posixpath to compose paths. So with `self.prefix = C:\some\path` we'll get something like `C:\some\path\/new/path`, which is not right.

The problem comes from the fact that we didn't actually ever think about connecting _to_ windows server and using ntpaths there. The scenario always was that you connect from windows(or any other client) to *nix host that uses normal posixpaths. I'm wondering if we should simply forbid ntpaths in ssh urls. Does mockssh(or, underlying paramiko server) support posixpaths when running on windows? If so, is there a way to create a temp dir and address it with posixpath in our tests?

Reply:
I see.

>Does mockssh(or, underlying paramiko server) support posixpaths when running on windows?

I think mockssh is designed to run not against any OS. So, maybe it is not support posixpaths when running on windows

>If so, is there a way to create a temp dir and address it with posixpath in our tests?

Let me check if we can address the temp dir with posixpath from `get_local_storagepath()`.

Do you have any suggestion to solve this problem?

Reply:
Windows actually supports `/` in paths, see https://en.wikipedia.org/wiki/Path_%28computing%29#MS-DOS.2FMicrosoft_Windows_style . So maybe something like `get_local_storagepath().replace('\\', '/')` would work.

Reply:
Cool. It solved the problem :)